channeldb/channel.go

```diff
@@ -532,6 +532,16 @@ type OpenChannel struct {
 	// for which we are the initiator.
 	FundingTxn *wire.MsgTx
 
+	// LocalShutdownScript is set to a pre-set script if the channel was opened
+	// by the local node with option_upfront_shutdown_script set. If the option
```

Comment:
this change should be moved to the commit where they are stored in the database :)

Reply:
Should I update  `LightningChannel.Local(/Remote)UpfrontShutdownScript` to just return empty addresses for this commit? Because I do like being able to spit this code from the DB code.